src/Darwin/KernelNotification.cs

```diff
@@ -247,14 +249,16 @@ public unsafe int KEvent (KernelEvent[] changeList, int nChanges, KernelEvent[]
 			return rv;
 		}
 
+// Don't worry about nullability for !NET
+#nullable disable
```

Comment:
I decided to leave this section as `#nullable disable` since it is !NET and would require that I either throw some ignore warnings statements, or check for not null arrays and that they have at least one item and possibly throw new errors/change implementation.